spec/features/items/edit_spec.rb

```diff
@@ -65,7 +65,7 @@
 
         click_button "Update Item"
 
-        expect(page).to have_content("Name can't be blank and Image can't be blank")
```

Comment:
If we could figure out the bug, then we'd add the `Image can't be blank` back again?

Reply:
No.. that was a test we were given that was later contradicted by the user story I was working on. So it's not related to a bug.